pytorch_lightning/metrics/functional/mean_relative_error.py

```diff
@@ -0,0 +1,54 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
Why not add this metric in a separate PR?

Reply:
no special reason

Reply:
well I need it here to measure regression

Reply:
As mentioned below, I think you can use `torch.testing.assert_allclose` and adjust the tolerance

https://github.com/pytorch/pytorch/blob/b9acfcddeb3b320f142f07d8051e5e344480f939/torch/testing/__init__.py#L209